packages/common/src/chains/sepolia.json

```diff
@@ -82,8 +82,8 @@
     },
     {
       "name": "mergeForkIdTransition",
-      "block": null,
-      "forkHash": null
+      "block": 1735371,
+      "forkHash": "0xb96cbd13"
```

Comment:
There is a comment above here "The forkHash will remain same as mergeForkIdTransition is post merge" but the fork hashes are not the same.